### PR TITLE
Bugfixx

### PR DESCRIPTION
**Pull request by Kev4ever**
Aub niet op naamgeving letten. Problemen met lock met andere naam.

### CODE REVIEW

File: code/InterestAR/Assets/Scripts/Services/MemoryDataService.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MemoryDataService
+{
+	private const string LANGUAGE = "language";
+	private const string DISTANCE = "distance";
+
+	public static string Language { 
+		get { return PlayerPrefs.GetString(LANGUAGE, "Englisch"); }
```
Comment (cyclops1007) [inline code/InterestAR/Assets/Scripts/Services/MemoryDataService.cs]: Englisch is not a language! Please change it to English.

**Review by jimvandeligt — CHANGES_REQUESTED** [review]
Code is up to standard. Spelling errors in the files SaveSystem.cs (PerisistencePath), MemoryDataService.cs (Englisch), SettingsMenu.cs (Englisch).

File: code/InterestAR/Assets/Scripts/SettingsMenu.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+    void Update()
+    {
+        ChangeRadiusVector();
+    }
+
+    private void ChangeRadiusVector()
+    {
+        float step = 1 + (slider.value / 2);
+        RadiusImage.rectTransform.localScale = new Vector2(step, step);
```
Comment (cyclops1007) [inline code/InterestAR/Assets/Scripts/SettingsMenu.cs]: Consider adding comments to possibly confusing code.

File: code/InterestAR/Assets/Scripts/SettingsMenu.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    // Values
+    private string language;
+    private float distance = 0;
+    private List<string> languages = new List<string> { "Nederlands", "Englisch", "Deutsch" };
+
+    // Gameobjects
+    public Button saveButton;
+    public Text distanceAmount;
+    public Dropdown languageDropdown;
+    public Slider slider;
```
Comment (cyclops1007) [inline code/InterestAR/Assets/Scripts/SettingsMenu.cs]: Edit to radiusSlider

File: code/InterestAR/Assets/Scripts/SettingsMenu.cs [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    public SVGImage RadiusImage;
```
Comment (cyclops1007) [inline code/InterestAR/Assets/Scripts/SettingsMenu.cs]: Edit to radiusImage

File: code/InterestAR/Assets/Scripts/settings/UserData.cs [hunk 5]
```diff
@@ -0,0 +1,22 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```
Comment (cyclops1007) [inline code/InterestAR/Assets/Scripts/settings/UserData.cs:4]: What is this file used for?

**Review by cyclops1007 — CHANGES_REQUESTED** [review]

File: code/InterestAR/Assets/Scripts/settings/UserData.cs (hunk 5 above)
Comment (Kev4ever) [reply code/InterestAR/Assets/Scripts/settings/UserData.cs:4]: We have discussed this

**Pull request merged**